docs/docs/intro.md

```diff
@@ -4,32 +4,42 @@ slug: /
 
 # Welcome to MarbleRun
 
-MarbleRun is a framework for creating distributed confidential computing apps.
+MarbleRun is a framework for deploying distributed confidential computing applications.
+MarbleRun acts as a confidential operator for your deployment. Think of a trusted party in the control plane.
 
-Build your confidential microservices with [EGo, Gramine, or another runtime](features/runtimes.md), distribute them with Kubernetes on an SGX-enabled cluster, and let MarbleRun take care of the rest. Deploy end-to-end secure and verifiable AI pipelines or crunch on sensitive big data in the cloud. Confidential computing at scale has never been easier.
+Build your confidential microservices with tools like [EGo, Gramine, or similar runtime](./features/runtimes.md), orchestrate them with Kubernetes on an SGX-enabled cluster, and let MarbleRun take care of the rest.
```

Comment:
You can either say "with tools like EGo and Gramine" or "with EGo, Gramine, or similar runtimes", but not both.